src/pages/traders/trader/index.js

```diff
@@ -127,14 +145,14 @@ function Trader() {
                             onClick={setSelectedTable.bind(undefined, 'spending')}
                         />
                     </ButtonGroupFilter>
-                    {trader.normalizedName !== 'fence' ? (
+                    {trader.normalizedName !== 'lightkeeper' ? (
                         <ButtonGroupFilter>
                             {trader.levels.map(level => (
                                 <ButtonGroupFilterButton
                                     key={level.level}
                                     tooltipContent={
                                         <>
-                                            {t('Unlocks at Loyalty Level {{level}}', { level: level.level})}
+                                            {t('Loyalty Level {{level}}', { level: level.level})}
```

Comment:
I was looking now at this on the website and I think before it was a more correct tooltip. When you filter by this it shows the items you will unlock on LLX, it's not showing all the items you have with the trader at LLX 🤔